index.js

```diff
@@ -2,8 +2,11 @@
 var objectAssign = require('object-assign');
```

Comment:
@sindresorhus here's a preview of the ES2015ification if you are ok with it cc @SamVerschueren

https://github.com/PedroFuentes/bitbar-npm/compare/5-add-submenus-support...PedroFuentes:5-add-submenus-support-es2015